protobuf/lib/src/protobuf/message_set.dart

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2011, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of '../../protobuf.dart';
+
+const _messageSetItemsTag = 1;
+const _messageSetItemTypeIdTag = 2;
+const _messageSetItemMessageTag = 3;
+
+abstract class $_MessageSet extends GeneratedMessage {
```

Comment:
We should add `/// nodoc` here otherwise it will appear in documentation.

Reply:
Yes please

Reply:
Also please add a (short) DartDoc explaining what this is.